src/assets/custom/css/_sass/_training-overview.scss

```diff
@@ -1,3 +1,7 @@
+$katalyst-banner-width: 384px;
+$katalyst-banner-height--small-medium: 313px;
```

Comment:
I think the height would come from having some padding top and bottom around the content, i.e. the copy and cta rather than being a set height. 

This doesn't need to be a blocker to getting merged though, once the copy comes through this can be adjusted.

Reply:
I agree, but don't think it should stop us from merging it. We'll need to make small adjustments when all the elements are in the page

Reply:
Sounds good, we do this when the translations come in :D